sale_cancel_reason/__manifest__.py

```diff
@@ -46,5 +20,5 @@
  'auto_install': False,
  'test': ['test/sale_order_cancel.yml',
```

Comment:
> (We won't use yml anymore ?)

Yes, I remember seeing several PR comments about this... @pedrobaeza, could you please provide any reference on this matter?